test/index.js

```diff
@@ -89,3 +123,12 @@ describe('Klaviyo', function () {
     });
   });
 });
+
+/**
+ * Decode base64 and parse json
+ */
+
+function decode(data){
+  var buf = new Buffer(data, 'base64');
+  return JSON.parse(buf.toString());
+}
```

Comment:
no eof newline, add [`ensure_newline_at_eof_on_save`](http://sublime-text-unofficial-documentation.readthedocs.org/en/latest/reference/settings.html#automatic-behavior) to your Sublime config